rasa/cli/evaluate.py

```diff
@@ -128,8 +133,16 @@ def _run_markers(
             "A file with the stats filename already exists"
         )
 
-    tracker_loader = _create_tracker_loader(endpoint_config, strategy, count, seed)
+    domain = Domain.load(domain_path)
     markers = Marker.from_path(config)
+
+    if not markers.validate_against_domain(domain):
```

Comment:
nice idea to just "bubble up" the warnings and then decide to break it here to gather all warnings first

Reply:
I was hoping to do it more like a compiler where you can track line number and also collect warnings a bit more elegantly, but it seemed like adding that would involve a lot of complicated stuff around adding information about where markers were parsed from to the markers and I didn't want to do too much in the time we have left